.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
+dist/
```

Comment:
📣 I think you need to do a `git rm -rf dist` as my understanding is that currently the dist folder is still checked in. Here you just stop us to update it...

Reply:
You're absolutely right. I thought gitignoring would be sufficient. Thank you.